ipet/evaluation/IPETEvalTable.py

```diff
@@ -1094,6 +1116,10 @@ def validateData(self, df : DataFrame) -> DataFrame:
             v = Validation(self.validate)
         else:
             v = Validation(None)
+        if self.feastol:
+            v.set_feastol(self.feastol)
```

Comment:
I would prefer to explicitly cast the `self.feastol` member to a float. in both calls. They might be strings for the evaluation object, if passed via xml.

Reply:
I agree, maybe do that in the setters already instead of the calls?

Reply:
No, not necessarily. This is simpler for exporting from and to xml to assume that all members of the evaluation are strings.